src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    private boolean startWorking;
+    private boolean stopWorking;
+
+    public Excavator(boolean startWorking, boolean stopWorking) {
+        this.stopWorking = stopWorking;
+        this.startWorking = startWorking;
+    }
```

Comment:
Why did you create these two variables and constructor for them? Please do not add things that are not specified in the task description. You should also remove `if` checks inside `stopWork` and `doWork`.